index.html

```diff
@@ -503,66 +504,64 @@
+Such <dfn>principles</dfn> describe the ways in which different [=actors=] may, must,
 or must not produce or [=process=] flows of information from, to, or about other [=actors=]
-([[?GKC-Privacy]], [[?IAD]]).
+([[?GKC-Privacy]], [[?IAD]]). Typically, <dfn>actors</dfn> are [=people=] but they can also be collective entities such as
+companies, associations, or governmental bodies, which are known as [=parties=].
 
-Typically, <dfn>actors</dfn> are [=people=] but they can also be collective endeavours such as
-companies, associations, or political parties (eg. in the case of espionage), which are known as
-[=parties=]. It is important to keep in mind that not all people are equal in how they can resist
+It is important to keep in mind that not all people are equal in how they can resist
 the imposition of unfair [=principles=]: some [=people=] are more [=vulnerable=] and therefore in greater
-need of protection. The focus of this document is on the impact that this power differential can
-have against people, but it can also impact other [=actors=], such as companies or governments.
+need of protection. The focus of this document is on the impact that differences in information power can
+have against people, but they can also impact other [=actors=], such as companies or governments.
 
 [=Principles=] vary from [=context=] to [=context=] ([[?Understanding-Privacy]], [[?Contextual-Integrity]]): people
 have different expectations of [=privacy=] at work, at a café, or at home for instance. Understanding and
 evaluating a privacy situation is best done by clearly identifying:
 
 * Its [=actors=], which include the subject of the information as well as the sender and the recipient
   of the [=information flow=]. (Note that recipients might not always want to be recipients.)
-* The specific type of data in the [=information flow=].
-* The [=principles=] that are in use in this specific context.
+* The type of data involved in the [=information flow=].
+* The [=principles=] that are in use in this context.
 
 It is important to keep in mind that there are <em>always</em> privacy [=principles=] and that all
-of them imply different power dynamics. Some [=principles=] may be more permissive, but that does
-not render them neutral — it merely indicates that they are supportive of the power dynamic that
-emerges from permissive [=processing=]. We must therefore determine which [=principles=]
+of them imply different power dynamics. Some sets of [=principles=] may be more permissive, but that does
+not make them neutral — it means that they support the power dynamic that
+comes with permissive [=processing=]. We must therefore determine which [=principles=]
 best align with ethical Web values in Web [=contexts=] ([[?ETHICAL-WEB]], [[?Why-Privacy]]).
 
-<dfn data-lt='information'>Information flows</dfn> as understood in this document are information
+<dfn data-lt='information'>Information flows</dfn> as understood in this document mean information
```

Comment:
```suggestion
<dfn data-lt='information'>Information flows</dfn> as used in this document refer to information
```
I'm not sure which is best among "are", "mean", and "refer to".